translations/ja.json

```
@@ -454,57 +454,57 @@
-	"Profile name \"{{.profilename}}\" is reserved keyword. To delete this profile, run: \"{{.cmd}}\"": "",
-	"Profile name '{{.name}}' is duplicated with machine name '{{.machine}}' in profile '{{.profile}}'": "",
-	"Profile name '{{.name}}' is not valid": "",
-	"Profile name '{{.profilename}}' is not valid": "",
-	"Profile name should be unique": "",
-	"Provide VM UUID to restore MAC address (hyperkit driver only)": "MAC アドレスを復元するための VM UUID を指定します（hyperkit ドライバのみ）",
-	"Pull images": "",
-	"Pull the remote image (no caching)": "",
-	"Pulling base image ...": "イメージを Pull しています...",
-	"Push images": "",
-	"Push the new image (requires tag)": "",
-	"Reboot to complete VirtualBox installation, verify that VirtualBox is not blocked by your system, and/or use another hypervisor": "",
-	"Rebuild libvirt with virt-network support": "",
-	"Received {{.name}} signal": "",
-	"Recreate the cluster by running:\n\t\tminikube delete {{.profileArg}}\n\t\tminikube start {{.profileArg}}": "",
-	"Registries used by this addon. Separated by commas.": "",
-	"Registry addon with {{.driver}} driver uses port {{.port}} please use that instead of default port 5000": "",
-	"Registry mirrors to pass to the Docker daemon": "Docker デーモンに渡すレジストリ ミラー",
-	"Reinstall VirtualBox and reboot. Alternatively, try the kvm2 driver: https://minikube.sigs.k8s.io/docs/reference/drivers/kvm2/": "",
-	"Reinstall VirtualBox and verify that it is not blocked: System Preferences -\u003e Security \u0026 Privacy -\u003e General -\u003e Some system software was blocked from loading": "",
-	"Related issue: {{.url}}": "",
-	"Related issues:": "",
+	"Print just the version number.": "バージョン番号だけ表示します。",
+	"Print the version of minikube": "minikube バージョンを表示します",
+	"Print the version of minikube.": "minikube のバージョンを表示します。",
+	"Problems detected in {{.entry}}:": "{{.entry}} で問題を検出しました:",
+	"Problems detected in {{.name}}:": "{{.name}} で問題を検出しました:",
+	"Profile \"{{.cluster}}\" not found. Run \"minikube profile list\" to view all profiles.": "「{{.cluster}}」プロファイルが見つかりません。全プロファイルを表示するために「minikube profile list」を実行してください。",
+	"Profile name \"{{.profilename}}\" is reserved keyword. To delete this profile, run: \"{{.cmd}}\"": "プロファイル名「{{.profilename}}」は予約語です。このプロファイルを削除するためには、「{{.cmd}}」を実行します",
+	"Profile name '{{.name}}' is duplicated with machine name '{{.machine}}' in profile '{{.profile}}'": "プロファイル名 '{{.name}}' は '{{.profile}}' プロファイル中のマシン名 '{{.machine}}' と重複しています",
+	"Profile name '{{.name}}' is not valid": "プロファイル名 '{{.name}}' は無効です",
+	"Profile name '{{.profilename}}' is not valid": "プロファイル名 '{{.profilename}}' は無効です",
+	"Profile name should be unique": "プロファイル名は単一でなければなりません",
+	"Provide VM UUID to restore MAC address (hyperkit driver only)": "MAC アドレスを復元するための VM UUID を指定します (hyperkit ドライバーのみ)",
+	"Pull images": "イメージを取得します",
+	"Pull the remote image (no caching)": "リモートイメージを取得します (キャッシュなし)",
+	"Pulling base image ...": "イメージを取得しています...",
```

```suggestion
	"Pulling base image ...": "ベースイメージを取得しています...",
```
Is it more accurate with "ベース"?